bin/picca_fast_metal_dmat.py

```diff
@@ -0,0 +1,567 @@
+#!/usr/bin/env python
+"""Compute the auto and cross-correlation of delta fields for a list of IGM
+absorption.
+
+This module follow the procedure described in sections 4.3 of du Mas des
+Bourboux et al. 2020 (In prep) to compute the distortion matrix
+"""
+import sys,os
+import time
+import argparse
+import numpy as np
+import fitsio
+
+from picca import constants, cf, utils, io
+from picca.utils import userprint
+
+
+def read_stack_deltas_table(filename) :
```

Comment:
Docstring missing. A simple line with 
```
"""
Read stack of deltas

Args:
    filename: str
        Name of the file to read
"""
```
should do

Reply:
done